torch/linalg/__init__.py

```diff
@@ -140,6 +140,66 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.norm(input, p=None, *, out=None) -> Tensor
+
+Returns the condition number of a matrix.
```

Comment:
Does input actually have to be a matrix? Does it have to be square?

This first sentence should define "condition number" OR be immediately followed by a mathematical statement describing it that includes input and p.

The first sentence should also reference input and p. For example: "Computes the condition number :attr:\`\input\`, a matrix, using the norm defined by :attr:\`p\`. The condition number is defined as: "

Reply:
Thank you for the suggestion. I've changed the text now.